integrations/web-chat/introduction.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Webchat and the Chat API"
+sidebarTitle: "Overview"
+description: "Use the PolyAI Chat API to power Webchat."
+---
+
+<Tip>Visit the [API reference](/api-reference/chat) for more detailed information on Chat API response parameters and authentication.</Tip>
+
+Use the Chat API to integrate PolyAI agents into existing <Tooltip tip="Non-voice chat systems, including live in-browser chats and widgets, as well as text and email conversations.">Webchat</Tooltip>
+systems, and use its endpoints to get programmatic access to your automated chat interactions. This API structures non-voice web
+integration conversations, messages, and chat sessions using a simple set of REST endpoints.
+
+The Chat API powers multiple PolyAI Webchat integrations:
+
+- [Email](email)
+- [SMS](sms)
+- [In-app chat](in-app-chat)
+- [WhatsApp](whatsapp)
+
+## Chatting with the API
+
+1. **Initiate conversation**: Send a POST `/chat/create` request with required headers.
+2. **Send and receive messages**: Use the `/chat/respond` endpoint for ongoing interaction.
+3. **Validation**: Requests are validated for API key, token, and payload.
+4. **Conversation termination**: Use the `end_conversation` flag in responses to identify when a session ends.
```

Comment:
Are we elaborating on these in a separate page?

Reply:
Also if we're moving to a separate page then AWS do this sort of thing quite well. They define a problem then walk through the solution with diagrams, code blocks and lots of explanation.
e.g. https://aws.amazon.com/blogs/contact-center/make-your-amazon-connect-chat-experience-more-engaging-with-custom-participants-and-generative-ai-powered-chatbots/

They then have separate pages also for each API endpoint (which the solution/example docs all link back to) which detail the request and response https://docs.aws.amazon.com/connect/latest/APIReference/API_CreateParticipant.html